### PR TITLE
Support NULLS LAST and NULLS FIRST syntax

### PR DESCRIPTION
**Pull request by kangkaisen**
Currently, when  select * from table order by a,b,c desc,  the NULL value will be first. In real application, the users maybe want to the NULL value always be last or first. So，we should support NULLS LAST and NULLS FIRST grammar like Impala and Oracle.

In fact, Doris BE has supported NULLS LAST and NULLS FIRST, we only need to make Doris FE support NULLS LAST and NULLS FIRST.

### CODE REVIEW

**Comment by kangkaisen** [comment]
Update this commit

**Review by imay — CHANGES_REQUESTED** [review]
Can you modify your commit message? You can find recommended format [here](http://doris.incubator.apache.org/guides/contributing.html)

**Comment by kangkaisen** [comment]
I have updated the commit message, please review, thanks.

File: fe/src/main/cup/sql_parser.cup [hunk 1]
```diff
@@ -205,9 +205,9 @@ terminal String KW_ADD, KW_ADMIN, KW_AFTER, KW_AGGREGATE, KW_ALL, KW_ALTER, KW_A
     KW_INNER, KW_INSERT, KW_INT, KW_INTERVAL, KW_INTO, KW_IS, KW_ISNULL,  KW_ISOLATION,
     KW_JOIN,
     KW_KEY, KW_KILL,
-    KW_LABEL, KW_LARGEINT, KW_LEFT, KW_LESS, KW_LEVEL, KW_LIKE, KW_LIMIT, KW_LINK, KW_LOAD, KW_LOCAL, KW_LOCATION,
+    KW_LABEL, KW_LARGEINT, KW_LAST, KW_LEFT, KW_LESS, KW_LEVEL, KW_LIKE, KW_LIMIT, KW_LINK, KW_LOAD, KW_LOCAL, KW_LOCATION,
     KW_MAX, KW_MAX_VALUE, KW_MERGE, KW_MIN, KW_MIGRATE, KW_MIGRATIONS, KW_MODIFY,
-    KW_NAME, KW_NAMES, KW_NEGATIVE, KW_NO, KW_NOT, KW_NULL,
+    KW_NAME, KW_NAMES, KW_NEGATIVE, KW_NO, KW_NOT, KW_NULL, KW_NULLS,
```
Comment (imay) [inline fe/src/main/cup/sql_parser.cup]: can you add KW_LAST an KW_NULLS to keyword grammar? 

If you don't add, old query may failed if there SQL contain these two words

Comment (kangkaisen) [reply fe/src/main/cup/sql_parser.cup]: I think I had added  KW_LAST an KW_NULLS to keyword grammar. Did I miss something?

Comment (imay) [reply fe/src/main/cup/sql_parser.cup]: You should add these here. 

// Keyword that we allow for identifiers
keyword ::=
    KW_AFTER:id
    {: RESULT = id; :}
    | KW_AGGREGATE:id
    {: RESULT = id; :}
    | KW_AUTHORS:id

**Review by imay — APPROVED** [review]
Good Job

**Pull request merged**